lib/cartopy/tests/crs/test_lambert_conformal.py

```diff
@@ -64,6 +65,45 @@ def test_specific_lambert():
                                   '+lat_1=35 +lat_2=65 +no_defs'))
 
 
+class Test_LambertConformal_standard_parallels(unittest.TestCase):
+    def test_single_value(self):
+        crs = ccrs.LambertConformal(standard_parallels=[1.])
+        assert_equal(crs.proj4_init, ('+ellps=WGS84 +proj=lcc +lon_0=-96.0 '
+                                      '+lat_0=39.0 +x_0=0.0 +y_0=0.0 '
+                                      '+lat_1=1.0 +no_defs'))
+
+    def test_no_parallel(self):
+        with self.assertRaisesRegexp(ValueError, '1 or 2 standard parallels'):
+            ccrs.LambertConformal(standard_parallels=[])
+
+    def test_too_many_parallel(self):
+        with self.assertRaisesRegexp(ValueError, '1 or 2 standard parallels'):
+            ccrs.LambertConformal(standard_parallels=[1, 2, 3])
+
+    def test_single_spole(self):
+        s_pole_crs = ccrs.LambertConformal(standard_parallels=[-1.])
+        assert_array_almost_equal(s_pole_crs.x_limits,
+                                  (-19840440, 19840440.),
+                                  decimal=0)
+        assert_array_almost_equal(s_pole_crs.y_limits,
+                                  (-370239953, -8191953),
+                                  decimal=0)
+
+    def test_single_npole(self):
+        # Take a look with:
+#        import matplotlib.pyplot as plt
+#        n_pole_crs = ccrs.LambertConformal(standard_parallels=[1])
+#        ax1 = plt.axes(projection=n_pole_crs)
+#        ax1.coastlines(); ax1.set_global()
```

Comment:
These comments cause PEP8 errors in the test suite, they should be a single block I guess.